index.html

```diff
@@ -0,0 +1,539 @@
+  <title>Smashing Magazine</title>
+  <link rel="stylesheet" href="style.css">
+  <script src="https://kit.fontawesome.com/e7a0251b7f.js" crossorigin="anonymous"></script>
+
+</head>
+
+<body>
+  <header>
+    <nav>
+      <div class="navlogo">
+        <a href="#">
+          <img class="smash" src="images/logo.svg" alt="">
+        </a>
+      </div>
+      <div class="navdiv">
+        <a href="#">
+          <div>
+            <img class="navicons transformer" src="images/articles.svg" alt="">
+          </div>
+          <div>
+            <h1>Articles</h1>
+            <h4>Design & development</h4>
+          </div>
+        </a>
+      </div>
+      <div class="navdiv">
+        <a href="#">
+          <div>
+            <img class="navicons transformer" src="images/books.svg" alt="">
+          </div>
+          <div>
+            <h1>Books</h1>
+            <h4>Physical & digital books</h4>
+          </div>
+        </a>
+      </div>
+      <div class="navdiv">
+        <a href="#">
+          <div>
+            <img class="navicons transformer" src="images/events.svg" alt="">
+          </div>
+          <div>
+            <h1>Events</h1>
+            <h4>Conferences & workshop</h4>
+          </div>
+        </a>
+      </div>
+      <div class="navdiv">
+        <a href="#">
+          <div>
+            <img class="navicons transformer" src="images/jobs.svg" alt="">
+          </div>
+          <div class="adiv">
+            <h1>Jobs</h1>
+            <h4>Find work & employees</h4>
+          </div>
+        </a>
+      </div>
+      <div class="navdiv">
+        <a href="#">
+          <div>
+            <img class="navicons transformer" src="images/membership.svg" alt="">
+          </div>
+          <div>
+            <h1>Membership</h1>
+            <h4>Webinars & early-birds</h4>
+          </div>
+        </a>
+      </div>
+      <div class="navsearch">
+        <a href="#">
+          <i class="fas fa-search"></i>
+          <span>Topics</span>
+        </a>
+      </div>
+    </nav>
+  </header>
+  <main>
+    <div class="container">
+      <div class="latest">
+        <p>Don't Miss These Latest</p>
+        <p>Articles</p>
+      </div>
+      <div class="serious-red">
+        <div>
+          <svg class="transformer" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" viewBox="0 0 24 24"
+            enable-background="new 0 0 24 24">
+            <g>
+              <path
+                d="m13.3 4.1c-.4-.1-.9-.1-1.3-.1-4.4.0-8 3.6-8 8s3.6 8 8 8c.4.0.9.0 1.3-.1 3.9-.6 6.7-4 6.7-7.9s-2.8-7.2-6.7-7.9zm-1.1 13.9c-.1.0-.1.0-.2.0-3.3.0-6-2.7-6-6s2.7-6 6-6c.1.0.1.0.2.0v12z">
+              </path>
+            </g>
+          </svg>
+        </div>
+        <div>
+          <p>Seriously, red?</p>
+        </div>
+      </div>
+      <div class="vitaly authors">
+        <div class="img-name">
+          <div><img class="persons transformer" src="images/vitaly-friedman.jpg" alt=""></div>
```

Comment:
There is no need to include images in your project

Reply:
Done!